### ISSUE TITLE
[Left 4 Dead 2 Beta - Linux] Achievements menu has letters for scroll buttons

### ISSUE THREAD

**Issue by stevethepocket** [ValveSoftware/Source-1-Games#554]
![2013-05-10_00001](https://f.cloud.github.com/assets/3085348/486729/3702b210-b934-11e2-9174-b2759f0e3b4a.jpg)

Probably a case of trying to use Marlett.ttf and not finding it. If you'd like, I can install the font manually off my Windows drive and see if that fixes the problem.

**Comment by vincent-t** [comment]
@gdrewb-valve @johndrinkwater This issue is still present. Any chance you guys can fix this?

**Comment by gdrewb-valve** [comment]
You'll have to wait for the team to comment.  If anything changes they'll update this issue.

**Comment by raku-cat** [comment]
Is there any plan to fix this ever or is this going down with every other linux l4d2 issue and never going to be fixed.

**Comment by peon501** [comment]
Speaking more abut symbols... ingame symols of keyboard keys also invisible :)

**Comment by My-Random-Thoughts** [comment]
For anyone that finds this issue and still has issues, you need to find and install the **Marlett** font TTF file.
Once you have the TTF file, copy it to your `~/.fonts` folder.

**Comment by stevethepocket** [comment]
This isn't a tech support thread. It's a "make Valve aware of this problem so they fix it" thread. In fact, it's better that people _not_ apply their own fixes because then they won't be able to spot when it's been fixed for real.

**Comment by My-Random-Thoughts** [comment]
While I agree with you, it's not been fixed in almost 10 years.  :)

**Comment by nm004** [comment]
> For anyone that finds this issue and still has issues, you need to find and install the **Marlett** font TTF file. Once you have the TTF file, copy it to your `~/.fonts` folder.

Thanks for the quick fix. You can also put the font file in `[Game Folder]/left4dead2/resource`. 

The font file can be obtained [the Wine repo](https://github.com/wine-mirror/wine/blob/7feaa6795a26c97b5ce4327a23129eb20a049859/fonts/marlett.ttf).

I think the game should include Marlette font as other Source 1 games, such as Half-Life 2 do.

**Comment by Tiagoquix** [comment]
Hi OP, is this still relevant as of 2024? I believe fonts were updated in L4D2 due to updates to improve Steam Deck (Linux) support. If possible, please test using Vulkan (add `-vulkan` to your launch options).

**Comment by nm004** [comment]
> Hi OP, is this still relevant as of 2024? I believe fonts were updated in L4D2 due to updates to improve Steam Deck (Linux) support. If possible, please test using Vulkan (add `-vulkan` to your launch options).

Manual fix is still needed (placing Marlett font in the desired directory).

**Comment by Tiagoquix** [comment]
@nm004 Are you using the native Linux version?

**Comment by nm004** [comment]
> @nm004 Are you using the native Linux version?

Does that mean the OpenGL version? I added the `-vulkan` option when I played it.

**Comment by Tiagoquix** [comment]
@nm004 Not necessarily. You can also play with emulated OpenGL and Vulkan in the Proton version. Please see if you are using the native or Proton, as that may influence the font bugs.

**Comment by nm004** [comment]
@Tiagoquix Only the native Linux version had the issue. I also run the game on Proton and the proper font were loaded (no need to place the font manually).

**Comment by Tiagoquix** [comment]
Thanks for the info!

**Comment by nm004** [comment]
FWIW, the game load the proper font when I install Marlett font into the user font directory (`~/.local/share/fonts`) and it is run on Steam Linux Runtime 1.0 Scout (the game can't run on 3.0 Sniper), although when the game run on the native platform, the game won't load the font from `~/.local/share/fonts`.
